From d357793b33ac6db8b66e83255ebf4aebd0ccddf8 Mon Sep 17 00:00:00 2001
From: "Philip, Avinash" <avinashphilip@ti.com>
Date: Tue, 6 Mar 2012 09:29:05 +0530
Subject: [PATCH 0815/1092] ARM: OMAP2+: am335x: Removes NAND device
 registration

Removes NAND device registration to make the way for new NAND device
registration API.

Signed-off-by: Philip, Avinash <avinashphilip@ti.com>
---
 arch/arm/mach-omap2/board-am335xevm.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/arch/arm/mach-omap2/board-am335xevm.c b/arch/arm/mach-omap2/board-am335xevm.c
index b505c88..82ed34e 100644
--- a/arch/arm/mach-omap2/board-am335xevm.c
+++ b/arch/arm/mach-omap2/board-am335xevm.c
@@ -1150,8 +1150,6 @@ static struct spi_board_info am335x_spi1_slave_info[] = {
 static void evm_nand_init(int evm_id, int profile)
 {
 	setup_pin_mux(nand_pin_mux);
-	board_nand_init(am335x_nand_partitions,
-		ARRAY_SIZE(am335x_nand_partitions), 0, 0);
 }
 
 /* TPS65217 voltage regulator support */
-- 
1.7.11.2

